src/Take.Blip.Builder/InputExpirationHandler.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Lime.Messaging.Contents;
+using Lime.Protocol;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Take.Blip.Builder.Models;
+using Take.Blip.Client.Content;
+using Take.Blip.Client.Extensions.Scheduler;
+
+namespace Take.Blip.Builder
+{
+    /// <summary>
+    /// Handle the logic of input expiration
+    /// </summary>
+    public class InputExpirationHandler : IInputExpirationHandler
+    {
+        private const string STATEID = "inputExpiration.stateId";
```

Comment:
Constant should be in pascal case

Reply:
Pascal case is to readonly or static, for constants is uppercase.

Reply:
:O bires is crazy